federatedscope/core/evaluator.py

```diff
@@ -41,22 +41,20 @@ def eval(self, ctx):
         results = {}
         y_true, y_pred, y_prob = self._check_and_parse(ctx)
         for metric, func in self.eval_metric.items():
-            results["{}_{}".format(ctx.cur_data_split,
-                                   metric)] = func(ctx=ctx,
-                                                   y_true=y_true,
-                                                   y_pred=y_pred,
-                                                   y_prob=y_prob,
-                                                   metric=metric)
+            results["{}_{}".format(ctx.cur_mode, metric)] = func(ctx=ctx,
```

Comment:
To be discussed: I am curious about how to get the results of "mode \times split"